compiler/aarch64/codegen/OMRTreeEvaluator.cpp

```diff
@@ -3155,11 +3223,7 @@ OMR::ARM64::TreeEvaluator::arraycopyEvaluator(TR::Node *node, TR::CodeGenerator
    TR::addDependency(deps, NULL, TR::RealRegister::x3, TR_GPR, cg);
    TR::addDependency(deps, NULL, TR::RealRegister::x4, TR_GPR, cg);
 
-   TR::SymbolReference *arrayCopyHelper = cg->symRefTab()->findOrCreateRuntimeHelper(TR_ARM64arrayCopy, false, false, false);
-
-   generateImmSymInstruction(cg, TR::InstOpCode::bl, node,
-                             (uintptr_t)arrayCopyHelper->getMethodAddress(),
-                             deps, arrayCopyHelper, NULL);
+   generateCallToArrayCopyHelper(node, srcAddrReg, dstAddrReg, lengthReg, deps, cg);
    cg->machine()->setLinkRegisterKilled(true);
```

Comment:
Please move the `setLinkRegisterKilled(true)` line into `generateCallToArrayCopyHelper()`.